doc/README.md

```diff
@@ -141,6 +141,17 @@ copy_to_distributed_table -CH -d '|' -n NULL input.csv users
 
 Call the script with the `-h` for more usage information.
 
+### Increasing INSERT throughput
+
+To maximize INSERT throughput, you should run statements in parallel. This helps utilizing multiple CPU cores. For instance, if you are loading data from two files you could run them in parallel such as the following:
+
+```
+copy_to_distributed_table -CH -d '|' -n NULL input_1.csv users &
```

Comment:
I've gotten into the habit of just copy-pasting from https://www.citusdata.com/blog/14-marco/164-parallel-data-loading-for-pg-shard whenever I need to load data into pg_shard. Would it be worth putting that script here? It seems more practical.